source/Halibut.Tests/TraceLogFileLogger.cs

```diff
@@ -1,25 +1,82 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Threading;
 
 namespace Halibut.Tests
 {
     public class TraceLogFileLogger : IDisposable
     {
-        string tempFilePath = Path.GetTempFileName();
+        readonly AsyncQueue<string> queue = new AsyncQueue<string>();
```

Comment:
Why not a `ConcurrentQueue`? We tend to use those more, so I'm wondering why we went this special type of queue that uses `VisualStudio` namespaces instead.

Reply:
No async methods :(

Reply:
Mmmm... This is only a test DLL not the actual product so it's not so bad, but I'm still fairly 👎  on overly-clever ConcurrentCollection type stuff.